jeremybarnes/jml
# Minor issues reported by clang compiler and Autotools test

Hi,

I just fixed a 3 minor issues in the arch. The clang compiler is reporting as error.

I am also adding the autotools configuration files.

It is tested with g++-4.7 and clang-3.2.1

To test it:

cd auto
./autogen.sh && ./configure && ./make check -j7

PS: Some unittests are failing
